Sources/App/routes.swift

```diff
@@ -11,66 +11,25 @@ public func routes(_ router: Router) throws {
     router.get("hello") { req in
         return "Hello, world!"
     }
-    
-    router.post(GithubWebhook.self, at: "git") { (req, webhook) -> Future<HTTPStatus> in
+   
+    router.post(GithubWebhook.self, at: "git") { (req, webhook) -> Future<Response> in
         guard
             let comment = webhook.comment,
             comment.user.login != "vapor-bot"
-            else { return req.future(.ok) }
-        let repo = webhook.repository
-        let issue = webhook.issue
-       
-        if comment.body.hasPrefix("@vapor-bot") {
-            let commands = comment.body.replacingOccurrences(of: "@vapor-bot ", with: "🔤").split(separator: "🔤")
-            let github = try req.make(GithubService.self)
-            guard commands.count >= 1 else {
-                return github.postComment(
-                    repo: repo.fullName,
-                    issue: issue.number,
-                    body: "Sorry? Didn't catch that.",
-                    on: req
-                ).transform(to: .ok)
-            }
-            let command = commands[0]
-            
-            switch command.lowercased() {
-            case "test performance":
-                let circle = try req.make(CircleCIService.self)
-                
-                guard let pullRequest = webhook.issue.pullRequest else {
-                    return github.postComment(
-                        repo: repo.fullName,
-                        issue: issue.number,
-                        body: "Unknown command",
-                        on: req
-                    ).transform(to: .ok)
-                }
-                
-                return try req.client().get(pullRequest.url).flatMap { response -> Future<String> in
-                    let pullRequestHead = try response.content.syncDecode(GithubPullRequest.self).head
-                    let branchName = pullRequestHead.name
-                    
-                    return circle.start(job: "linux-performance", repo: repo.fullName, branch: branchName, on: req)
-                }.flatMap { _ -> Future<CreateCommentResponse> in
-                    return github.postComment(
-                        repo: repo.fullName,
-                        issue: issue.number,
-                        body: "Starting performance test",
-                        on: req
-                    )
-                }.transform(to: .ok)
-                
-            default:
-                return github.postComment(
-                    repo: repo.fullName,
-                    issue: issue.number,
-                    body: "Unknown command",
-                    on: req
-                ).transform(to: .ok)
-            }
+            else { return try HTTPStatus.ok.encode(for: req) }
+        
+        let githubRouter = GithubCommandRouter()
+        try githubRoutes(router: githubRouter)
```

Comment:
I'd register the command router as a service so that it can be cached. That way it doesn't need to be initialized each time this route is called.